nym-vpn-core/crates/nym-vpn-lib/src/types.rs

```diff
@@ -0,0 +1,8 @@
+// Copyright 2025 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: GPL-3.0-only
+
+#[derive(Debug, Clone, Copy)]
+pub(crate) struct AvailableBandwidth {
```

Comment:
This `type.rs` seems a bit out of place. 
If it's only used by the BC, maybe define it there?
If it's gonna be used in multiple places, then `nym-vpn-lib-types` is the place

Reply:
Granted, `bandwidth_controller.rs` is a bit big, maybe a `bandwidth_controller/mod.rs`?